api/validation/__init__.py

```diff
@@ -0,0 +1,37 @@
+from functools import wraps
+
+from flask import request, Request
+from marshmallow import Schema, ValidationError
+
+from api.validation.schemas import EC2Action
+
+
+def __valid_request(_request: Request, *, json_schema: Schema = None, args_schema: Schema = None, cookies_schema: Schema = None):
```

Comment:
readability: would `validate_request` be a more representative name?

Reply:
I like it, but I'd still like to keep it "private" with the double underscore since as of now the function is not intended to be used elsewhere.
Coming up in next revision, thanks!